### DISCUSSION TITLE
Problema para llenar el formulario

### DISCUSSION THREAD

**Discussion by AdrianaOlivaBarba** [LaunchX-InnovaccionVirtual/CursoIntroPython#247] [Q&A]
Hola, una pregunta a alguien más le sale este mensaje cuando intenta llenar el fomulario, ya inicie sesión con la cuenta de innovación y aún así no puedo avanzar :(

![Captura de Pantalla 2022-02-14 a la(s) 4 17 27 p m](https://user-images.githubusercontent.com/92202840/153955791-1786cbd7-451b-4aa5-952c-7e1561ace36a.png)

**Comment by JosephLeyva** [comment]
Creo que es porque tienes que entrar con el correo de LaunchX

**Reply by AdrianaOlivaBarba** [reply]
Gracias, ya pude, muy amable :D

**Comment by doguedogue** [comment]
Comentó la sherpa que abrieras en una ventana de incognito y usaras tus credenciales de @innovaccion

**Reply by AdrianaOlivaBarba** [reply]
Gracias, así si pude, no había entendido esa parte muy amable :D
